lib/rqliteDialect.ts

```diff
@@ -127,6 +129,23 @@ export class RqliteDialect extends Client implements Client_ {
 
     return connection;
   }
+
+  _formatQuery(sql, bindings, timeZone) {
```

Comment:
Did you see, that knex moved their formatQuery to a different file?
https://github.com/knex/knex/blob/2aec21e/lib/execution/internal/query-executioner.js#L6

What do you think about importing the formatQuery from query-executioner.js instead of copying it here? By importing it, we could avoid having to maintain the port :).